### PR TITLE
BaSP-MR-302: remove add button in Member table from Admin

### PR DESCRIPTION
**Pull request by Emanuel-Lamberti**
The button to create a new member in the members table (accessing as Admin) was removed.

### CODE REVIEW

File: src/Components/Members/index.js [hunk 1]
```diff
@@ -40,10 +36,6 @@ function Members() {
 
   return (
     <section>
-      <div>
```
Comment (FrancoLelli) [inline src/Components/Members/index.js:43]: I think you should not delete the button because the administrator should use it

**Review by FrancoLelli — CHANGES_REQUESTED** [review]

**Review by CristianLotorto — APPROVED** [review]
Nice job Ema!

File: src/Components/Members/index.js (hunk 1 above)
Comment (Emanuel-Lamberti) [reply src/Components/Members/index.js:43]: According to the logic of the problem, the administrator should not create members, they can edit or delete them, but not create them.

**Review by FrancoLelli — APPROVED** [review]
Nice Job mate!

**Pull request merged**